### PR TITLE
v1.4 Fix User Guide Bugs

### PR DESCRIPTION
**Pull request by guohuang88**
- Update User Guide to remove all instances of address book
- Change erroneous commands

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -23,7 +23,7 @@ PatientSync is a **desktop app made for nurses to manage patient, optimized for
 
 1. Download the latest `PatientSync.jar` from [here](https://github.com/AY2324S2-CS2103-F09-2/tp/releases).
 
-1. Copy the file to the folder you want to use as the _home folder_ for your AddressBook.
+1. Copy the file to the folder you want to use as the _home folder_ for your PatientSync.
```
Comment (SherisseTJW) [inline docs/UserGuide.md]:
```suggestion
1. Copy the file to the folder you want to use as the _home folder_ for PatientSync.
```

File: docs/UserGuide.md [hunk 2]
```diff
@@ -222,7 +222,7 @@ Format: `delete INDEX`
 
 Examples:
 * `find Bernice` followed by `delete 1` deletes the 1st patient in the results of the `find` command.
-* `list` followed by `delete 1` deletes the 1st patient in the results of the address book.
+* `list` followed by `delete 1` deletes the 1st patient in the results of the `list` command.
```
Comment (SherisseTJW) [inline docs/UserGuide.md]:
```suggestion
* `list` followed by `delete 1` deletes the 1st patient based on the results of the `list` command.
```

**Review by SherisseTJW — CHANGES_REQUESTED** [review]
some minor edits for English, otherwise generally lgtm

File: docs/UserGuide.md (hunk 2 above)
Comment (guohuang88) [reply docs/UserGuide.md]: I agree that this phrasing is better!

File: docs/UserGuide.md (hunk 1 above)
Comment (guohuang88) [reply docs/UserGuide.md]: I agree that this phrasing is better!

**Review by SherisseTJW — APPROVED** [review]
lgtm

**Pull request merged**